### DISCUSSION TITLE
Release notes: How can I see what new/changed for Production?

### DISCUSSION THREAD

**Discussion by JurgenZiemer** [edmcouncil/fibo#1383] [General]
The release notes provide great highlights and in-depth detail for the Development version.
Are there similar release notes for Production or does the detail bullet point indicate whether the change was propagated from DEV to PROD?
Thanks

**Comment by ElisaKendall** [comment]
Hi Jurgen, 

They cover a combination of production and development.  We have not created notes that separate the two to date.  For the most part, the changes to BE, FBC, FND, IND, and SEC are production changes, and the others are mainly in development.

Best,

Elisa

**Comment by JurgenZiemer** [comment]
Thank you, Elisa,
For your fast response, clarifying my question.

**Comment by rivettp** [comment]
But I think this distinction is something to consider being more explicit about for future releases.

**Comment by JurgenZiemer** [comment]
I would like to raise Release Notes for Production and Documentation of the Configuration Management as a request.
Many uses case, including KG, depend on static PROD “vetted by SMEs and passed standard industry hygiene tests for OWL” rather than work in progress DEV.
To start with a list of ontologies, classes, and properties promoted from DEV to PROD with the release and backed out PROD elements. The list should refer to existing DEV release note bullet points.

Most Prod users will be shocked to find 2020/Q1 having more than four hundred fewer classes than 2019/Q4. 429 Benchmarks, e.g. fibo-ind-ir-cm:AED-EBOR-Reuters changed from class to named individual. The redesign makes sense, but it must be communicated in bold and red.

**Comment by rivettp** [comment]
Not quite clear of the detail of what you're requesting, @jziemer2 ...
- what do you mean by "Documentation of the Configuration Management";
- note that individual classes and properties don't get promoted, only whole ontologies. I don't think it's useful to list any more than the ontologies in the release notes;
- by "backed out PROD elements" do you mean those that were in PROD and deleted or deprecated? Note that I don't think we ever revert an ontology from PROD back to DEV.
- I don't understand "The list should refer to existing DEV release note bullet points". Is that for the list of ontologies promoted from DEV to PROD? Are you saying that change should also be in the DEV release notes?
- Given that DEV includes PROD, should the DEV release notes include all of the PROD notes (split across the domains) or should the DEV notes list only the changes in addition to those covered by the PROD notes? I prefer the latter to avoid duplication though it means people looking in 2 places for the full set of changes in a domain including DEV ontologies

**Comment by JurgenZiemer** [comment]
- Listing the Prod ontologies that were added or updated is a start.
- The easiest for the FIBO team and users is one list in a spreadsheet. Users can filter by the environment.

We have a lot of information about the development process, GitHub setup, and how to participate.
By configuration management, I mean the process of determining what goes into the Prod release, "vetted by SMEs," how ontologies promote, and the "standard industry hygiene tests for OWL."
